sdk/nanotdf_test.go

```diff
@@ -239,3 +240,89 @@ func NotTestCreateNanoTDF(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestGetECPublicKeyKid(t *testing.T) {
+	var tests = []struct {
+		name       string
+		kasURL     string
+		dialOption grpc.DialOption
+		shouldFail bool
+	}{
+		{
```

Comment:
It seems kinda overkill to have a table test with one row

Reply:
just keeping the same test pattern

Reply:
added two cases